services/datalinker/templates/NOTES.txt

```diff
@@ -0,0 +1,22 @@
+1. Get the application URL by running these commands:
```

Comment:
Delete this file; it's only used for manual Helm installations and doesn't do anything useful in an Argo CD environment.

Reply:
done